tests/test_decision_service.py

```diff
@@ -1320,9 +1319,6 @@ def test_get_variation_for_feature__returns_variation_for_feature_in_group(self)
             "group_exp_1", "28901"
         )
         with mock.patch(
-            "optimizely.decision_service.DecisionService.get_experiment_in_group",
-            return_value=(self.project_config.get_experiment_from_key("group_exp_1"), []),
-        ) as mock_get_experiment_in_group, mock.patch(
             "optimizely.decision_service.DecisionService.get_variation",
             return_value=(expected_variation, []),
         ) as mock_decision:
```

Comment:
This mock does not give a test for "group". It tests the cases for feature-tests not bucketed.
Can we mock bucket to control group-exclusion cases?